environment/interpretation/lane_detector/src/detector/inversePerspectiveTransform.cpp

```diff
@@ -70,7 +70,7 @@ void callbackFunc(int event, int x, int y, int flags, void* userdata) {
 }
 
 cv::Mat LaneDetector::inversePerspectiveTransform(cv::Mat &image) {
-    my_ipt_offsets_file = ipt_offsets_file;
+    my_ipt_offsets_file ="/home/kinoder/catkin_ws/src/freezing-batman/environment/interpretation/lane_detector/data/" +  ipt_offsets_file;
```

Comment:
This way it will only on your laptop, use paths relative to the project's home directory.